### ISSUE TITLE
Alternative Placement idea

### ISSUE THREAD

**Issue by calamitycons** [AriahSiwik/tuhc-rda#2]
I have two ideas for places to put the flash, one is more labor intensive than the other though.

Idea number 1: **Just after page 2590**, when Aradia skips to the end and begins her normal recap of the fight. This would be a good spot to show Aradia skipping to, where **she simply pops in, joins the fight,** and **then we skip over eight pages and hop to 2598**, with some **modified dialog from Aradia** opening up her internal chatlog for the first time and placing her normal recap dialogue there. **This is the more labor intensive option** that requires skipping over Aradia's original recap and replacing it with the actual fight.

Idea number 2: Immediately **after page 2597,** where Aradia stops recapping the fight and discusses what happened at the new universe door. On page 2597 itself, Aradia adds an extra line of dialog saying something like "**We have made a video recording** from multiple viewpoints as added documentation" so that it flows right into the fight. This is **less labor intensive but** runs the risk of being **repetitive** as it is right after the major-beats recap of the fight.

**Comment by ariahks** [comment]
Yeah, I thought about those! I couldn't think of a way to make the second option work well, but the first option could be a good idea. Not an option I'd recommend to new readers since it removes some content but I'd imagine it'd be as good a place as any.

I'll bring this up with my partner as well, but I'd need to come up with a name for the flash/command, and a way to nicely move on from that and into the "but ultimately..." dialog. I'd imagine the dialog on page 2590 can stay as is.

**Comment by calamitycons** [comment]
I think a good title might be “[S] Aradia: Engage Black King” or maybe “Join the Fight,” something like that.

**Comment by 123ldk** [comment]
I personally decided to plop it right between 2589 (That's what I like to hear.) and 2590 (Aradia: Skip to the end.)
This has the benefit of continuity- Aradia vanishes on the previous page, then reappears on this one.

If you name RDA's page (Aradia: Skip to the end.) and 2590 something like (Aradia: Recap.) or (Aradia: Start recap.) , you avoid having to modify her dialogue or skip over it, because the recap is supposed to come after the fight in the context of the comic in the first place, if that makes sense.

**Comment by HyperTem** [comment]
@calamitycons 
I decided to go out of my way and implement a similar placement idea to your first idea myself, on a fork of this mod. I also included a 1080p HD version of the animation with my update as a bonus. I'll take it down if requested by the original author, of course, but i'm assuming it won't cause any issues since this repo hasn't been updated in 3 years lol
Here it is!
https://github.com/HyperTem/tuhc-rda
